### DISCUSSION TITLE
Recognizing your contributions with digital badges

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#86] [General]
Background: A digital credential is a representation of skill achievements. When issued via the Open Badge system, it can be displayed, accessed and verified online via meta data that tethers to the Issuer, providing the reader various details behind the credential.

Digital credentials – such as badges – have become the industry standard skills currency in the marketplace. Many companies are increasingly adopting open badges to support their certification processes, internal career development and more.

We're currently exploring a couple of avenues with regards to issuing digital badges. Let's discuss some options in this discussion.

A couple of paid options:
* https://www.accredible.com/
* https://info.credly.com/ - IBM has a license to issue badges via credly

Questions for you:
* Are there open source badge issuers that we should consider?
* What kinds of badges would you like to see for z/OS Open Source?

**Comment by MikeFultonDev** [comment]
@IgorTodorovskiIBM now that we have a larger community, is this worth re-asking? Maybe on an upcoming guild call or perhaps a survey?
